app/assets/images/payment_icons/sezzle.svg

```diff
@@ -1 +1 @@
+<svg width="38" height="24" role="img" viewBox="0 0 38 24" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-sezzle"><title id="pi-sezzle">Sezzle</title><defs><linearGradient x1="80.524%" y1="85.62%" x2="45.753%" y2="37.68%" id="pi-sezzle-linearGradient-1"><stop stop-color="#CE5DCB" offset="0%"/><stop stop-color="#C558CC" offset="20.95%"/><stop stop-color="#AC4ACF" offset="55.25%"/><stop stop-color="#8534D4" offset="98.45%"/><stop stop-color="#8333D4" offset="100%"/></linearGradient><linearGradient x1="-.001%" y1="49.995%" x2="99.997%" y2="49.995%" id="pi-sezzle-linearGradient-2"><stop stop-color="#FF5667" offset="2.37%"/><stop stop-color="#FC8B82" offset="65.92%"/><stop stop-color="#FBA28E" offset="100%"/></linearGradient><linearGradient x1="-.003%" y1="50%" x2="100%" y2="50%" id="pi-sezzle-linearGradient-3"><stop stop-color="#00B874" offset="0%"/><stop stop-color="#29D3A2" offset="51.26%"/><stop stop-color="#53DFB6" offset="68.17%"/><stop stop-color="#9FF4D9" offset="100%"/></linearGradient><linearGradient x1="-.003%" y1="49.999%" x2="99.999%" y2="49.999%" id="pi-sezzle-linearGradient-4"><stop stop-color="#FCD77E" offset="0%"/><stop stop-color="#FEA500" offset="52.41%"/><stop stop-color="#FF5B00" offset="100%"/></linearGradient></defs><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g fill-rule="nonzero"><rect fill="#FFF" x="0" y="0" width="38" height="23.222" rx="3.167"/><g transform="translate(11 1)"><path d="M1.417 14.044a4.836 4.836 0 006.84 0l.18-.181c.944-.944-.945-5.896 0-6.84l-7.02 7.02z" fill="url(#pi-sezzle-linearGradient-1)"/><path d="M8.62 6.84l-.182.182c-.943.944.944 5.896 0 6.84L15.46 6.84a4.823 4.823 0 00-3.42-1.416A4.817 4.817 0 008.62 6.84" fill="url(#pi-sezzle-linearGradient-2)"/><path d="M1.417 7.204a4.836 4.836 0 000 6.84L8.62 6.84a4.836 4.836 0 000-6.84L1.417 7.204z" fill="url(#pi-sezzle-linearGradient-3)"/><path d="M8.256 14.044a4.836 4.836 0 000 6.84l7.204-7.205a4.836 4.836 0 000-6.84l-7.204 7.205z" fill="url(#pi-sezzle-linearGradient-4)"/></g></g></g></svg>
```

Comment:
It's missing the border around the icon. An example can be found inside the [contribution document](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

Reply:
Thanks @tauthomas01.  Should be fixed now.